pkg/api/plugins.go

```diff
@@ -469,6 +469,10 @@ func (hs *HTTPServer) InstallPlugin(c *contextmodel.ReqContext) response.Respons
 		if errors.Is(err, plugins.ErrInstallCorePlugin) {
 			return response.Error(http.StatusForbidden, "Cannot install or change a Core plugin", err)
 		}
+		var archError repo.ErrArchUnsupported
+		if errors.As(err, &archError) {
+			return response.Error(http.StatusConflict, archError.Error(), nil)
```

Comment:
Curious about the choice of status conflict. I think of it as not found, as you're asking Grafana to install a plugin, but we could not find a version that supports your architecture. I would expect a conflict if maybe I tried to install a version of a plugin that was already installed?

Reply:
I copied it from `versionUnsupportedErr` case above but I agree with the reasoning, it makes sense for this to return a 404 (maybe the `versionUnsupportedErr` is wrongly set?)

Reply:
Ah yes - good for pointing that out. We are currently able to tell the difference between when a version does not exist and when a version exists but does support your system. Maybe then for consistency between the two, this should actually be `ErrArchNotFound`?

Reply:
makes sense :+1: